src/main/java/scala_maven/ScalaMojoSupport.java

```diff
@@ -290,6 +290,13 @@ void addLibraryToClasspath(Set<File> classpath) throws Exception {
     }
   }
 
+  void addScalaDocToClasspath(Set<File> classpath) throws Exception {
+    Context sc = findScalaContext();
+    for (Artifact dep : sc.findScalaDocAndDependencies()) {
+      classpath.add(dep.getFile());
+    }
+  }
+
```

Comment:
Why do you place this method into this class (vs into `ScalaDocMojo.java`, the only place where it is called) ?